QUDAConfig.cmake.in

```diff
@@ -0,0 +1,41 @@
+@PACKAGE_INIT@
```

Comment:
This is a very basic config file. It sets some flags from the existing configuraiton, and so depending on those we can resolve dependencies.